docs/developers/0_preparation.md

```diff
@@ -0,0 +1,22 @@
+# 💼 Preparing a development environment
+In order to compile ReVanced Manager, certain requirements must be met.
+
+## 📝 Prerequisites
+* Kotlin IDE such as [Android Studio](https://developer.android.com/studio)
+* Knowledge of [Android](https://android.com) app development, and the [Kotlin](https://kotlinlang.org/) language.
+* At least JDK 17 of any vendors
+
+## 🏃 Prepare the environment
+1. Clone the repository
+   ```sh
+   git clone https://github.com/ReVanced/revanced-manager.git && cd revanced-manager
+   ```
+2. Build the APK
```

Comment:
`2. Build the application` or `2. Build the app`

Reply:
```suggestion
2. Build the application
```


Co-authored-by: Palm <palmpasuthorn@gmail.com>